botBackend/draft_logic.py

```diff
@@ -524,8 +541,80 @@ def reset(self):
         self.column = 2
         self.row_move = []
         self.column_move = []
-        self.picks_remaining = []
+        self.picks_remaining = 0
         self.snake_player_list = []
 
+        # backup the state of the draft to being reset
+        self.backup()
+
         # call sheet api to reset
         sheetapi.reset_sheet()
+
+    ###################################
+    ###      BACKUP DRAFT LOGIC     ###
+    ###################################
+
+    def backup(self):
```

Comment:
this looks pretty nasty and manual, I will admit. However, the main reason for this is that dataclasses are not JSON-ifiable so to speak. Because of this, I have to do a manual and yucky process to back things up. It's a bit gross, but what can you do.

Reply:
There is likely a better solution then what I have here, but I do not have the time to find it.